### DISCUSSION TITLE
蓝灯Windows版网速慢，不稳定，请及时处理我的请求

### DISCUSSION THREAD

**Discussion by Xisong-Feng** [getlantern/lantern-forum-cn#933] [General]
每次提交反馈问题给邮箱就没下文了，实在没精力解决就请给我退款吧

**Comment by MaxMKMax** [comment]
@Francis-ui-cmd 您好. 您提交问题反馈了, 给我那个 8 位数票号. 他们一般不会看邮箱反馈的. 需要我这里主动在开会给他们提问题. 因为太多太多邮件了. 他们忙不过来. 不好意思啊. 非常抱歉.

**Reply by Xisong-Feng** [reply]
77738430，不过邮件上显示the ticket has been closed

**Comment by Xisong-Feng** [comment]
请问处理的后续呢？不跟进就又没有下文了？
现在每次使用蓝灯都要花差不多半小时刷新、重连节点，搞得和玄学一样，后续使用也不稳定，用得好好的突然又断了，太影响使用体验了

**Comment by Xisong-Feng** [comment]
实在是没法用，请给我退款吧，到时候过一个月又不让退了

**Comment by Xisong-Feng** [comment]
支付宝付款的
订单号：2024100522001401431445257164
商家订单号：202410051244581144965824

**Comment by MaxMKMax** [comment]
@Francis-ui-cmd 您好. 已经帮您提交退款了. 请问您的账号邮箱是什么?

**Reply by MaxMKMax** [reply]
@Francis-ui-cmd  请问您的账号邮箱是什么?

**Reply by Xisong-Feng** [reply]
1225774778@qq.com

**Reply by greySkiesMarsh** [reply]
Hello @Francis-ui-cmd, I'm very sorry that resolving this has taken so long. 

Unfortunately, we have been migrating to a new payment system and I wasn't able to find your transaction using the details that you've provided here. If possible, could you reply with a screenshot of your email receipt? If you aren't able to upload it onto this forum, you can send it to me directly at marsh@getlantern.org.

**Reply by Xisong-Feng** [reply]
> Hello @Francis-ui-cmd, I'm very sorry that resolving this has taken so long.
> 
> Unfortunately, we have been migrating to a new payment system and I wasn't able to find your transaction using the details that you've provided here. If possible, could you reply with a screenshot of your email receipt? If you aren't able to upload it onto this forum, you can send it to me directly at [marsh@getlantern.org](mailto:marsh@getlantern.org).

![webwxgetmsgimg](https://github.com/user-attachments/assets/fe88d60a-d580-412c-a737-307424617d78)

**Reply by Xisong-Feng** [reply]
> Hello @Francis-ui-cmd, I'm very sorry that resolving this has taken so long.
> 
> Unfortunately, we have been migrating to a new payment system and I wasn't able to find your transaction using the details that you've provided here. If possible, could you reply with a screenshot of your email receipt? If you aren't able to upload it onto this forum, you can send it to me directly at [marsh@getlantern.org](mailto:marsh@getlantern.org).

我邮箱没有收到收据，只有支付宝的支付凭证

**Comment by MaxMKMax** [comment]
@Francis-ui-cmd 您好. 支付宝单据也可以了. 请您发到这里给我一下. 谢谢您.

**Reply by Xisong-Feng** [reply]
> @Francis-ui-cmd 您好. 支付宝单据也可以了. 请您发到这里给我一下. 谢谢您.

![webwxgetmsgimg](https://github.com/user-attachments/assets/0fbe3f35-f5a2-4e86-95b2-0d91bad6c946)

**Reply by Xisong-Feng** [reply]
> @Francis-ui-cmd 您好. 支付宝单据也可以了. 请您发到这里给我一下. 谢谢您.

麻烦了！

**Reply by Xisong-Feng** [reply]
> @Francis-ui-cmd 您好. 支付宝单据也可以了. 请您发到这里给我一下. 谢谢您.

您好，请尽快给我处理，因为马上要到购买后一个月了，我不希望到时候退不了

**Comment by MaxMKMax** [comment]
@Francis-ui-cmd 您好. 已经帮您提交好退款申请了. 我再帮您提醒一下. 谢谢您的耐心等待.

**Reply by Xisong-Feng** [reply]
好的谢谢！我现在再等待一段时间

**Reply by Xisong-Feng** [reply]
> @Francis-ui-cmd 您好. 已经帮您提交好退款申请了. 我再帮您提醒一下. 谢谢您的耐心等待.

您好，现在已经一个半月过去了，退款仍然没有消息。
